packages/viz/src/lib/Layer.ts

```diff
@@ -213,6 +223,23 @@ export class Layer implements StyledLayer {
       await this._replaceLayer();
     }
   }
+
+  public remove() {
+    if (this._deckInstance === undefined) {
+      throw new CartoLayerError(
+        'Cannot remove because it was not attached to map',
```

Comment:
I think we should put `This layer cannot be removed because it is not added to a map`.

Just tense changes and using same "language" as we have in our methods 😁

Reply:
Done